starter-code/index.html

```diff
@@ -0,0 +1,195 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <link rel="stylesheet" href="../starter-code/css/main.css" />
+    <title>REVERA</title>
+    <!-- Bootstrap CSS -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta/css/bootstrap.min.css" integrity="sha384-/Y6pD6FV/Vv2HJnA6t+vslU6fwYXjCFtcEpHbNJ0lyAFsXTsjBbfaDjzALeQsN6M" crossorigin="anonymous">
+  </head>
+
+  <body>
+    <header class="nav-bar">
+      <nav class="navbar navbar-expand-lg navbar-light bg-light">
+      <div class="collapse navbar-collapse" id="navbarNav">
+        <h1>REVERA</h1>
+        <ul class="navbar-nav">
+          <li class="nav-item active">
+            <a class="nav-link" href="#">Home <span class="sr-only">(current)</span></a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="#">About</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="#">Sample page</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link disabled" href="#">Portfolio</a>
+          </li>
+          <li class="nav-item">
+          <a class="nav-link" href="#">Blog</a>
+          </li>
+          <li class="nav-item">
+          <a class="nav-link" href="#">Parent page</a>
+          </li>
+          <li class="nav-item">
+          <a class="nav-link" href="#">Contact</a>
+          </li>
+        </ul>
+      </div>
+    </nav>
+    </header>
+    <div class="carousel-caption d-none d-md-block text-slide">
+      <h3>FUSCE SCELERISQUE NIBH AC NISL LUCTUS INTERDUM.</h3>
+      <p>Quisque sit amet sagittis justo. Suspendisse ornare, felis feugiat suscipit pharetra,
+        lorem est viverra odio; tristique tristique diam tellus id tellus.
+        In consectetur aliquet tristique. Curabitur imperdiet leo in nisl consectetur mattis.
+        Quisque bibendum, mauris vitae suscipit ornare; est nibh elementum mauris,
+        eu malesuada nunc magna at justo. Proin blandit id tellus non pellentesque. Mauris […]</p>
+    </div>
+    <div id="carouselExampleControls" class="carousel slide" data-ride="carousel">
+      <div class="carousel-inner" role="listbox">
+        <div class="carousel-item active">
+          <img class="d-block img-fluid" src="../starter-code/img/slider.jpg" alt="First slide">
```

Comment:
When linking resources like images, it is important to use the simplest relative pass as the src. This src will only work locally, but not on a server. 

In this case, it should be: src="img/slider.jpg" 
This means "in the current folder, find the folder 'img', and then find the file 'slider' " 

Your code **src="../starter-code/img/slider.jpg"** means: 
"from current folder, go up one folder, then find folder 'starter-code', find folder 'img', find file 'slider' ". This doesn't work on a server, when the code is executed already on the root directory, making it look like the screenshot below.

![image](https://user-images.githubusercontent.com/29508784/30318879-9b6cd7dc-97ae-11e7-8c95-e32565f26b3e.png)

Reply:
Don't worry though, I was able to view the proper version of your work when I opened it locally on my computer :)